src/Components/Subscriptions/index.js

```diff
@@ -1,9 +1,216 @@
-import styles from './subscriptions.module.css';
+import React, { useEffect, useState } from 'react';
+import style from './subscriptions.module.css';
 
 function Subscriptions() {
+  const [subscriptions, setSubscriptions] = useState([]);
+  const [create, setCreate] = useState({
+    classes: '',
+    member: '',
+    date: ''
+  });
+  const [showForm, setShowForm] = useState(false);
+  const [button, setButton] = useState('');
+
+  const getSubscriptions = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/subscriptions`, {
```

Comment:
you need to change here with ${process.env.REACT_APP_API_URL}/api/subscription for vercel deploy

Reply:
done